components/event-bus/internal/knative/subscription/controller/subscription/reconcile.go

```diff
@@ -86,16 +89,31 @@ func (r *reconciler) Reconcile(request reconcile.Request) (reconcile.Result, err
 				log.Error(err, "SetReadySubscription() failed for the subscription:", "subscription", subscription)
 				reconcileErr = err
 			} else {
+				//TODO READY Subscription
+				log.Info("Ready Subscription!")
```

Comment:
Please make all the monitoring logs shown in `DEBUG`; otherwise, there will be too many logs that we wouldn't be able to see the root cause of a problem.

Reply:
Also, what does this mean? What do I need to understand from `Ready Subscription!`? You can make it more clear what's happening.

Reply:
Right, this was missed doing cleaning the code. Will remove them.
Thanks